c/progress/progress_manager.h

```diff
@@ -28,7 +28,7 @@ class progress_bar_enabled;
 class work;
 
 
-enum class InterruptStatus : unsigned char {
+enum InterruptStatus : unsigned char {
```

Comment:
Why don't use the `enum class` here?

Reply:
Because we're assigning it to a `sig_atomic_t` variable, and if `InterruptStatus` was a class, we'd need to insert explicit type-casts everywhere.

Reply:
Oh, I see, `interrupt_status` is now just a `sig_atomic_t` that basically makes it an integer?..

Reply:
Yeah, I already found it, for some reason github showed me your comment only now...